addons/mail/static/src/new/rtc/call_settings.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+
+<t t-name="mail.settings" owl="1">
```

Comment:
Do you use no indentation for templates?

Reply:
i use copy paste :). there is no tooling, so i don't care too much, honestly